clients/fides-js/custom-rollup-plugins/prevent-double-load.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Rollup plugin to prevent double loading of Fides scripts
+ */
+export const preventDoubleLoad = (globalName) => {
+  const multipleLoadingMessage = `${globalName} detected that it was already loaded on this page, aborting execution! See https://www.ethyca.com/docs/dev-docs/js/troubleshooting for more information.`;
+
+  // Common replacement pattern for all UMD wrappers
+  const doubleLoadCheck = `if(typeof ${globalName}!=="undefined" && ${globalName}.options && ${globalName}.options.fidesUnsupportedRepeatedScriptLoading !== "enabled_acknowledge_not_supported"){console.error("${multipleLoadingMessage}");return;}`;
```

Comment:
I'd recommend changing this check a bit. It currently expects `Fides.options` to exist but that seems unnecessary.

I'd do something more akin to this:
```typescript
if (typeof Fides !== "undefined") {
   // Bad news! Multiple loading detected!
   // In the special case where this is allowed, continue, but otherwise...
   if (Fides.options && Fides.options.fidesUnsupportedRepeatedScriptLoading == "enabled_acknowledge_not_supported") {
     // no-op, continue
  } else {
    console.error("bad!");
  }
}
```

Does this make sense? I am struggling to communicate this

Reply:
I double checked, and our minified `fides.js` is already using optional chaining operators (`foo?.bar`) so I should be safe to just use that here as well.